### PR TITLE
Record new/updates images

### PR DESCRIPTION
**Pull request by MarcoEidinger**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

**Comment by MarcoEidinger** [comment]
Images recorded with Xcode 12.1

**Review by billzhou0223 — APPROVED** [review]

**Comment by MarcoEidinger** [comment]
Waiting for merge until we plan to merge necessary changes in SAP/cloud-sdk-ios-fiori to execute snapshot tests with Xcode 12.1

**Comment by MarcoEidinger** [comment]
Dropping for now

**Pull request closed without merging**
